aalto/jupyterhub.rst

```diff
@@ -262,6 +262,12 @@ FAQ and bugs
   and eventually needs to be cleaned up - after the respective course
   is done.
 
+* **I don't see the assignments for my course**.  There are different
```

Comment:
Maybe this should be little more precise on what to do. Either some direct instructions on how to change spawners (i.e "On the upper right corner, click "Control Panel" next to "Logout", you should arrive to new page with either one or two buttons. First click "Stop my Server", wait a little bit and click "Start My Server" and choose a different spawner")

Or just step by step guide with pictures:
![image](https://user-images.githubusercontent.com/6150958/88069939-c6cd1880-cb7a-11ea-9cd2-aa04b126c5d1.png)
![image](https://user-images.githubusercontent.com/6150958/88070004-d9475200-cb7a-11ea-8de1-6b9a9e5d9199.png)
etc.